version.go

```diff
@@ -0,0 +1,6 @@
+package taskq
+
+// Version is the current release version.
+func Version() string {
+	return "0.0.0"
```

Comment:
should we specifc the version as the tag version , eg `3.2.4`

Reply:
This will be filled automatically by release.sh script